cypress/support/utils.js

```diff
@@ -131,13 +128,16 @@ export const fillInNewGroupedLabelForm = (
 };
 
 export const addNewLabelToGroup = (groupName, labelName, condition, rank) => {
+
   // Find tab with the group name and click on it
   cy.get('button[data-baseweb="tab"')
     .should("exist")
     .find('div[data-testid="stMarkdownContainer"]')
     .should("exist")
     .contains(groupName)
-    .click();
+    .as("labelGroupTab");
```

Comment:
we might want to refactor this to use the as() earlier. sometimes, by the time the test reaches contains() there has been a rerender, so it's better to make an alias earlier on. I haven't looked at these tests yet, but if this point is flaky then that could be one thing we could try.

Reply:
yes, you are absolutely correct! I've already refactored couple of calls like this for probes I believe.